### DISCUSSION TITLE
Is there a way that Pipeline continues if any one of its previous jobs passes

### DISCUSSION THREAD

**Discussion by silbabai0110** [concourse/concourse#7168] [Help & Support]
We have scenario where we need to have a build deployed in higher env (lets say stage env) if only one of it's previous jobs (viz. QA or INT) has passed.
Putting **passed: [DEPLOY-QA, DEPLOY-INT]** in DEPLOY-STG job wont work as it would be an AND instead of an OR, both DEPLOY-QA & DEPLOY-INT job needs to pass to proceed with DEPLOY-STG job.
Is this use case possible in concourse currently? If YES how can we achieve it.
Can anyone please help with this scenario.

**Comment by crsimmons** [comment]
AFAIK there is no native way to accomplish that in Concourse. I think the closest you could get is having the earlier jobs update some external resource (such as a file in S3) when they succeed then triggering the STG job off of that resource. You might have to do something to make sure STG doesn't get triggered twice but that sounds solvable.
